### ISSUE TITLE
Not found in play store

### ISSUE THREAD

**Issue by keraliss** [btcontract/wallet#196]
the [play store](https://play.google.com/store/apps/details?id=com.btcontract.wallet) link for the wallet is not working anymore. is the app removed?

**Comment by akumaigorodski** [comment]
Yes, Google has removed it, I had not yet time to figure out why, will fix it eventually.

**Comment by endingisnight** [comment]
Good time to add a fdroid [badge](https://gitlab.com/fdroid/artwork/blob/master/badge/README.md) to the readme. Or did you not know you're there as well? https://f-droid.org/packages/com.btcontract.wallet/

**Comment by akumaigorodski** [comment]
Not any more, their requirements are too restrictive for my future plans, I'm not sure why since version of it is still there, they have promised to removed it since last update.

**Comment by endingisnight** [comment]
> Not any more, their requirements are too restrictive for my future plans, I'm not sure why some version of it is still there, they have promised to remove it since last update.

I cant find an issue on fdroiddata about removing the app, if you asked for it in the matrix chat or forum they might've understandably forgotten. But what are your plans and why isn't fdroid a part of them? Are they propriatary?

**Comment by akumaigorodski** [comment]
https://github.com/akumaigorodski/wallet/issues/191

**Comment by akumaigorodski** [comment]
You have asked about the nature of an issue, I have kindly responded with details, that is all I intended to do.
